build.rs

```diff
@@ -34,8 +36,19 @@ const FLAGS: &[&str] = &[
     "-lwxexpat",
 ];
 
+fn save_pkg_path(pkg_path: &str) {
+    let out_dir = env::var_os("OUT_DIR").unwrap();
+    let dest_path = Path::new(&out_dir).join("pkg_path.rs");
+    fs::write(
+        &dest_path,
+        format!("static PKG_PATH: &str = r\"{}\";", pkg_path)
+    ).unwrap();
+    println!("cargo:rerun-if-changed=build.rs");
+}
+
 fn main() {
     let pkg_path = env::var("CARGO_MANIFEST_DIR").unwrap();
+    save_pkg_path(&pkg_path);
```

Comment:
Saving path filled flags to generated source reduces duplicated flags definition.

Reply:
resolved in 471d45fe30d45aa2d8b218a47e36b7e718ef6076